### DISCUSSION TITLE
I accidentally removed editor role from teleport-admin

### DISCUSSION THREAD

**Discussion by PavelNiedoba** [gravitational/teleport#13405] [Q&A] [answered]
Is it possible to recover or I need to start over.  I guess that all servers will permanently loose connections.

**Comment by webvictim** [comment, marked as answer]
If you have `root` access to the Teleport auth server, you can run this set of commands locally on that machine to give yourself the `editor` role back.

1) Download the user definition to a YAML file:
```console
sudo tctl get user/teleport-admin > teleport-admin.yaml
```

2) Edit the `teleport-admin.yaml` file and add `editor` back to the list of roles in the file:
```yaml
spec:
  roles:
  - access
  - auditor
  - editor
```

3) Recreate the `teleport-admin` user from the file:
```console
sudo tctl create -f teleport-admin.yaml
```

This should give you back your `editor` permissions. Your connected nodes should be unaffected.
